src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -1206,8 +1175,7 @@ private MenuBar createMenu() {
                 view,
                 tools,
                 options,
-                help
-        );
```

Comment:
@Siedlerchr Just a minor point: Is it possible to disable this particular auto-format of the braces in eclipse? I noticed similar reformattings also in other PRs of you. In my opinion, there are situation where it the code is easier to understand if the braces are on the next line (when it functions more as a grouping then as  a function argument)

Reply:
Hm., I will try to have a look at this.